external_file_location/models/task.py

```diff
@@ -138,6 +138,7 @@ def _template_render(self, template, record):
     def run_task_scheduler(self, domain=None):
         if domain is None:
             domain = []
+        domain.append([('enabledd', '=', True)])
```

Comment:
Are you sure the domain is correct ?
For me there is too many levels.

It should be  : domain.append(('enabled', '=', True)) no ?

Reply:
merci